src/Platform/Webservices/Connector.php

```diff
@@ -1526,10 +1526,127 @@ public function dutchBusinessGetDossierHistory($dossierNumber, $periodStartDate,
             [
                 'dossier_number' => $dossierNumber,
                 'period_start_date' => $periodStartDate,
-                'period_end_date' => $periodEndDate, ]
+                'period_end_date' => $periodEndDate,
+            ]
         );
     }
 
+    /**
+     *  Starts a UBO investigation.
+     *
+     * @param string      $dossierNumber   chamber of Commerce number
+     * @param string|null $oldestExtractDate Period start date, in Y-m-d format [optional]
+     * @param bool        $useUpdates   Use a real-time extract [optional]
+     *
+     * @return \stdClass <DutchBusinessUBOInvestigationToken>
+     */
```

Comment:
Maybe you can add \@link annotation to the documentation? (also for the other dutchBusiness methods)